contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -132,10 +143,9 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
             // In the wrapped token case, we know that the bridgeToken to be burned
             // is not a "fee/burn on transfer" token, since it was deployed by this
             // contract itself.
-            require(
-                totalAmount > primaryFeeAmount + secondaryFeeAmount,
-                "Fee amounts more than total."
-            );
+            if (totalAmount <= primaryFeeAmount + secondaryFeeAmount) {
+                revert FeeAmountsMoreThanTotal(totalAmount, primaryFeeAmount + secondaryFeeAmount);
```

Comment:
nit: fee amount can be equal to total but the name is FeeAmountsMoreThanTotal.

Reply:
if the fee totalAmount equals total fees, then I think we transfer 0 amount.